src/app/routes/index.js

```diff
@@ -4,7 +4,11 @@ import MediaPage from '../containers/MediaPage';
 import getArticleInitialData from './getInitialData/article';
 import getFrontpageInitialData from './getInitialData/frontpage';
 import getMediaPageInitialData from './getInitialData/mediapage';
-import { articleRegexPath, frontpageRegexPath, mediaRegexPath } from './regex';
+import {
+  articleRegexPath,
+  frontpageRegexPath,
+  mediaRadioRegexArray,
```

Comment:
Missed TV

Reply:
This made me realise that I should combine the radio and tv output into a single array as they for the immediate future will share Component and params